packages/@aws-cdk/custom-resource-handlers/lib/custom-resources-framework/callable-expr.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { $E, Expression, IScope, ThingSymbol, expr } from '@cdklabs/typewriter';
+
+/**
+ * A class representing an expression proxy which builds a JavaScript object that
+ * will mirror the JavaScript operations done to it in an expression tree.
+ */
+export class CallableExpr {
```

Comment:
This is a comment for almost all the files in this folder. I'm not seeing a lot of tests here. I'd like to see more unit tests added.

Reply:
Additional unit tests added.